.ci-builds/.azure-pipelines.yml

```diff
@@ -58,6 +58,18 @@ jobs:
       inputs:
         filename: icu4c/source/allinone/icucheck.bat
         arguments: 'x64 Release'
+    - task: PowerShell@2
+      displayName: 'PowerShell: Distrelease script'
+      inputs:
+        targetType: filePath
+        filePath: 'icu4c/packaging/distrelease.ps1'
+        arguments: '-arch x64'
+        workingDirectory: icu4c
```

Comment:
Just checking, does this mean the script will be run from the `icu4c` directory?

Reply:
Right. It effectively does a `cd` to this directory before running the script, just like the instructions say to do. :)